lib/engine/step/dividend.rb

```diff
@@ -88,13 +89,31 @@ def payout(entity, revenue)
         { company: 0, per_share: payout_per_share(entity, revenue)[0] }
       end
 
+      def dividends_for_entity(entity, holder, per_share)
+        return 0 if (percent = holder.percent_of(entity)).zero?
```

Comment:
can you just do num_shares_of

Reply:
Sure, although looking at the code I see that `num_shares_of` just calls `percent_of` and divides by 10.

Reply:
Actually, when I changed `percent_of` to `num_shares_of` a ton of unit tests failed. I can look into why tomorrow.

Reply:
Never mind, I see what your point was.